### DISCUSSION TITLE
Login and forgot password with custom field

### DISCUSSION THREAD

**Discussion by bocanhcam** [laravel/nova-issues#6346] [Ideas & Feature Requests] [answered]
Some of my applications will have an ID or username beside an email for users.

Is it possible to reset or log in with a username or ID instead of an email?

**Comment by crynobone** [comment, marked as answer]
Released with Laravel Nova 5.0.0

You should be able to customize by using `usernameUsing()` and `emailUsing()`.

```php
    /**
     * Register the configurations for Laravel Fortify.
     */
    protected function fortify(): void
    {
        Nova::fortify()
            ->usernameUsing('username')
            ->register();
    }
```
